### DISCUSSION TITLE
Pi Pico W - rp2.country('AU') - still don't see channel 13

### DISCUSSION THREAD

**Discussion by gmrza** [micropython/micropython#10332] [RP2040 / Pico]
I am having trouble connecting to an AP running on channel 13, even after setting rp2.country('AU').  Has anybody else been having this problem?
As soon as I set the AP in question to channel 11, my Pi Pico W can see it.

My code currently looks as follows:
```
rp2.country('AU')
wlan = network.WLAN(network.STA_IF)
wlan.active(True)
# Turn off wireless power saving
wlan.config(pm = 0xa11140)
wlan.connect(ssid, password)
```

**Comment by gmrza** [comment]
Even after the official release of Pico W support with v1.20, I am still not managing to get the Pico W to use channel 13.

> I am having trouble connecting to an AP running on channel 13, even after setting rp2.country('AU'). Has anybody else been having this problem? As soon as I set the AP in question to channel 11, my Pi Pico W can see it.
> 
> My code currently looks as follows:
> 
> ```
> rp2.country('AU')
> wlan = network.WLAN(network.STA_IF)
> wlan.active(True)
> # Turn off wireless power saving
> wlan.config(pm = 0xa11140)
> wlan.connect(ssid, password)
> ```

**Comment by guruthree** [comment]
I'm seeing a similar problem - unable to connect to 802.11n 2.4 GHz Channel 12 or 13 with country code set to GB on the Pico W with MicroPython 1.22.2.

**Comment by scruss** [comment]
Maybe try `network.country()` instead of `rp2.country()`

**Reply by oleksii-leonov** [reply]
I have checked, `rp2.country("XX")` automatically sets the `network.country()` to "XX", so it's not the issue.

**Comment by 7ka-Hiira** [comment]
I am using https://github.com/earlephilhower/arduino-pico and I am having a very similar issue where I can't use channel 13 when I set the WiFi Region to `Japan` or `Australia`. It was available when I set it to `Worldwide`.
